server/controller/websocket.py

```diff
@@ -98,37 +98,32 @@ def log_dump(message, websocket, metadata):
     log data, parses it and sends the parsed data to all connected web clients.
 
     Args:
-        message: The decoded JSON message from the Mobile API.
-        websocket: The WebSocket connection object where the log is being
-            received.
+        message: the decoded JSON message from the Mobile API
+        websocket: the full websocket connection
+        metadata: the metadata object for the WebSocket connection
     """
-    print('logs sent')
-    print(message)
-
-    # TODO: (Sumner) fix when implementing the iOS parsing component.
-    if metadata['osType'] == 'iOS':
-        return
-
-    parsed_logs = LogParser.parse(message)
+    log_entries = list(
+        LogParser.parse(message['rawLogData'], metadata['osType'])
+    )
 
     api_key = metadata.get('apiKey', '')
 
-    associated_websockets = (
-        controller.user_management_interface.find_associated_websockets(api_key,
-            _web_ui_ws_connections))
-
-    # Send to database and convert to html.
-    html_logs = LogParser.convert_to_html(parsed_logs['logEntries'])
+    # Send to database and convert to HTML.
     controller.datastore_interface.store_logs(
         metadata['apiKey'], metadata['deviceName'], metadata['appName'],
-        metadata['startTime'], metadata['osType'], parsed_logs)
+        metadata['startTime'], metadata['osType'], log_entries)
 
+    # Create a message to send to the web clients.
     send_logs = {
         'messageType': 'logData',
-        'osType': 'Android',
-        'logEntries': html_logs,
+        'osType': metadata['osType'],
+        'logEntries': LogParser.convert_to_html(log_entries),
     }
 
+    # Determine wich WebSocket connections to send the logs to
+    associated_websockets = controller.user_management_interface \
```

Comment:
Does this also fit with the style guide? I'm pretty sure we're supposed to use parentheses for doing long line continuations rather than the \ operator.